loky/backend/context.py

```diff
@@ -291,6 +269,60 @@ def _count_physical_cores():
     return cpu_count_physical, exception
 
 
+def _count_physical_cores_linux():
+    try:
+        cpu_info = subprocess.run(
+            "lscpu --parse=core".split(), capture_output=True, text=True
+        )
+        cpu_info = cpu_info.stdout.splitlines()
+        cpu_info = {line for line in cpu_info if not line.startswith("#")}
+        return len(cpu_info)
+    except:
+        pass  # fallback to /proc/cpuinfo
```

Comment:
The fallback code is not covered by the tests. Would it be possible to somehow hide "lscpu" (or even delete/rename) in the VM of the Linux CI hosts?

Reply:
monkey-patch subprocess.run maybe so that it fails?

Reply:
do you think ``lscpu`` is still needed ? i.e. are there systems where ``lscpu`` exists but not ``/proc/cpuinfo`` ?

Reply:
No idea, but better be safe than sorry.